go-controller/pkg/node/node.go

```diff
@@ -521,6 +521,39 @@ func (n *OvnNode) WatchEndpoints() {
 	}, nil)
 }
 
+func (n *OvnNode) validateMTU(mtu int) error {
```

Comment:
Three things:
1. This needs a docblock
2. Naming suggestion: `validateGatewayMTU`
3. Pass just the interface name, instead of the MTU?